content/en/03-using-pixie/01-using-live-ui.md

```diff
@@ -79,8 +79,39 @@ To learn how to use Pixie scripts for a specific use case, check out one of the
 
 To learn how to write your own PxL script, check out the [tutorial](/tutorials/pxl-scripts/write-pxl-scripts).
 
-For writing quick, one-off scripts, use the `Scratch Pad`. The `Scratch Pad` is the first item in the drop-down `script` menu. Open the script editor to add code to the empty PxL Script.
+When writing custom scripts, we recommend starting from an existing script which does something similar to the result you are looking for.
 
-<svg title='Use the Scratch Pad script .' src='live-ui/live-ui-7.png'/>
+### Edit an Existing Script
 
-The `Scratch Pad` does not yet support persistence, so if you are extensively developing a script, we recommend using the [Script Developer Environment](/tutorials/pxl-scripts/script-dev-environment).
+1. Select the script from the `script` drop-down menu in the top left.
+2. Open the script editor using the keyboard shortcut: `ctrl+e` (Windows, Linux) or `cmd+e` (Mac).
+3. Make edits to the script.
+4. Run the script using the `RUN` button in the top right or by using the keyboard shortcut: `ctrl+enter` (Windows, Linux) or `cmd+enter` (Mac).
+5. Hide the script editor using `ctrl+e` (Windows, Linux) or `cmd+e` (Mac).
+
+### Use the Scratch Pad
+
+Use the `Scratch Pad` to:
+
+- Develop quick, one-off scripts.
+- Run example scripts from the [`pixie-demos`](https://github.com/pixie-io/pixie-demos) repository.
+
+The `Scratch Pad` is the first item in the `script` drop-down menu:
+
+<svg title='The Scratch Pad is the first item in the script menu.' src='live-ui/scratch-pad.png'/>
+
+When you select the `Scratch Pad`, the script editor will open. The script editor contains two tabs:
```

Comment:
feels like this applies both to the editing scripts and using the scratch pad section